tensorflow_serving/g3doc/setup.md

```diff
@@ -62,11 +62,11 @@ sudo apt-get update && sudo apt-get install -y \
 ### Clone the TensorFlow Serving repository
 
 ~~~shell
-git clone --recurse-submodules https://github.com/tensorflow/serving
+git clone --recursive https://github.com/tensorflow/serving
 cd serving
 ~~~
 
-`--recurse-submodules` is required to fetch TensorFlow, gRPC, and other
+`--recursive` is required to fetch TensorFlow, gRPC, and other
```

Comment:
Could you please also update all other instances of recurse-submodules?

tensorflow_serving/g3doc/docker.md 
tensorflow_serving/g3doc/serving_inception.md